apps/site/lib/site_web/views/bus_stop_change_view.ex

```diff
@@ -110,7 +72,9 @@ defmodule SiteWeb.BusStopChangeView do
     if(stop, do: link(stop.name, to: stop_path(conn, :show, stop.id), class: "text-primary"))
   end
 
-  @spec time_range(%Alert{}) :: Phoenix.HTML.Safe.t()
+  @spec time_range(%Alert{} | %HistoricalAlert{}) :: Phoenix.HTML.Safe.t()
+  def time_range(%HistoricalAlert{alert: alert}), do: time_range(alert)
```

Comment:
question: should this and `alert_item/1` below also be a part of the `AmbiguousAlert` interface?